test/server.js

```diff
@@ -1,42 +0,0 @@
-var express = require('express');
```

Comment:
server.js was a fake server used for the acceptance tests, but it's not really needed. We have removed it from other templates too